spec/api/houdini/nonprofit_spec.rb

```diff
@@ -144,7 +144,7 @@ def h(h = {})
 
       #expect(Houdini::V1::Nonprofit).to receive(:sign_in)
 
-      xhr :post, '/api/v1/nonprofit', input
+      post '/api/v1/nonprofit', params: input, xhr: true
```

Comment:
xhr and the use of positional params in request specs is deprecated so this corrected that.